contracts/common/Vesting/Vesting.sol

```diff
@@ -46,7 +46,7 @@ abstract contract Vesting is IVesting, Governed, APRCalculatorConnector {
     /**
      * @inheritdoc IVesting
      */
-    function setPenaltyDecreasePerWeek(uint256 newRate) external onlyRole(DEFAULT_ADMIN_ROLE) {
+    function setPenaltyDecreasePerWeek(uint256 newRate) external onlyGovernance {
```

Comment:
At this point we can't apply contract changes without using upgrade, so this PR must include test and docs changes only

Reply:
Will do this changes in a separated PR